flink/v1.12/flink/src/main/java/com/netease/arctic/flink/read/hybrid/enumerator/AbstractArcticEnumerator.java

```diff
@@ -68,7 +68,7 @@ public void close() throws IOException {
   }
 
   @Override
-  public void handleSplitRequest(int subtaskId, @Nullable String requesterHostname) {
+  public void handleSplitRequest(int subtaskId, @Nonnull String requesterHostname) {
```

Comment:
This is opposite annotation. Original means field can be null, but this Nonnull means can not be null.

Reply:
yes, I will fix it